spec/descriptions/getSyntheticTests.md

```diff
@@ -0,0 +1,27 @@
+This endpoint retrieves Synthetic Tests.
+
+## Optional Parameters:
+
+- **locationId** Filters the Synthetic Tests to retrieve only the ones that are associated to the specified PoP location ID.
+- **applicationId** Filters the Synthetic Tests to retrieve only the ones that are associated to the specified Application Perspective Unique Id. 
```

Comment:
```suggestion
```
Since we are going to introduce applicationIds, and applicationId is also supported in filter part, probably we should not promote it here.

Reply:
agreed